tests/patmat/i7186.scala

```diff
@@ -172,16 +172,16 @@ object printMips {
 
     def oneAddr[O]
       ( a: O, indent: String)
-      ( r: O => Dest,
+      ( r: a.type => Dest,
```

Comment:
I imagine that the changes to this file can be avoided

Reply:
I reverted the changes